test/integration/index.js

```diff
@@ -0,0 +1,3 @@
+require('./effectsLevelTest');
+require('./hourOfCodeTest');
+require('./spriteDanceTest');
```

Comment:
It looks like the tests aren't cleaning up after themselves, when running in the same context?  Probably non-blocking, but would be good to add a `teardown` step so only the currently-running visualization is visible.  We do something similar [in Craft integration tests](https://github.com/code-dot-org/craft/blob/b16dc2939cc30990c0134238f41b1d1c55664451/test/helpers/RunLevel.js#L38-L42).

![screen shot 2018-11-09 at 12 22 12 pm](https://user-images.githubusercontent.com/413693/48286520-c0d0eb80-e41a-11e8-844c-92a5a3ecea43.png)

Reply:
Fixed in commit 6b44343082509995e0afdb1483cdbe0066bebf87.